sdk/keyvault/keyvault-admin/src/generated/keyVaultClient.ts

```diff
@@ -0,0 +1,327 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ *
+ * Code generated by Microsoft (R) AutoRest Code Generator.
+ * Changes may cause incorrect behavior and will be lost if the code is
+ * regenerated.
+ */
+
+import * as coreHttp from "@azure/core-http";
+import * as Models from "./models";
+import * as Mappers from "./models/mappers";
+import * as Parameters from "./models/parameters";
+import * as operations from "./operations";
+import { KeyVaultClientContext } from "./keyVaultClientContext";
+
+class KeyVaultClient extends KeyVaultClientContext {
```

Comment:
@joheredi Is there a way we can specify the name of the client we want to use when generating the code?

Reply:
Yes there is. I don't think it would make sense to rename this client though, since this client does not fully wrap the whole set of functionalities the generated code is supporting. The convenience layer is really what matters, whenever we decide to do that.

Reply:
that makes sense, thanks!